### PR TITLE
Mage Tower

### PR DESCRIPTION
**Pull request by terrapin47**

### CODE REVIEW

File: QUEST.tsv [hunk 1]
```diff
@@ -651,15 +651,15 @@ QUEST_20150317_000650	Will of All Things
 QUEST_20150317_000651	The base forms of all things are Water, Earth, Wind, and Fire, {nl}and these are controlled by the Goddess.
 QUEST_20150317_000652	Elementary Fire Magic
 QUEST_20150317_000653	{memo X}1. Dried environment{nl}2. Something combustible. {nl}3. Enough mana. {nl}4. The blessing of Goddess Gabija.
-QUEST_20150317_000654	The diary of Agailla Flurry
+QUEST_20150317_000654	The Diary of Agailla Flurry
```
Comment (Sunaries) [inline QUEST.tsv]: Agailla Flurry's Diary sounds better.

File: ETC.tsv [hunk 2]
```diff
@@ -5022,7 +5022,7 @@ ETC_20150317_005021	{@st41}Open your inventory by pressing the {img F2 40 40} ke
 ETC_20150317_005022	{memo X}The Mausoleum lanterns are being attacked by the guardians!
 ETC_20150317_005023	{memo X}Bearkaras' suppressed power has returned.
 ETC_20150317_005024	Panto_Rsword: Hot Panto Fighter: 50/Panto_Rspear: Hot Panto Spearman: 50/Panto_Rhand: Hot Panto: 50/Panto_Rstaff: Hot Panto Wizard: 51
-ETC_20150317_005025	$Crafting the item
+ETC_20150317_005025	$Crafting
```
Comment (Sunaries) [inline ETC.tsv:5025]: Why did you change this?

Comment (terrapin47) [inline ETC.tsv:5025]: Always seems silly when it comes up, since it's kind of redundant.  What else would you be crafting, but the item?

Comment (vtange) [inline ETC.tsv:5025]: i personally think it looks cooler when it tells me i'm crafting _the thing_ i've been trying to get

Comment (terrapin47) [inline ETC.tsv:5025]: Like if it was a Crafting %s ?  I'd agree.

**Comment by Sunaries** [comment]
Good to merge,

**Comment by imcgames** [comment]
:+1:
